packaging/packager_rpm.sh

```diff
@@ -12,30 +12,33 @@ dnf -y install python3 python3-coverage
 dnf -y install python3-pillow
 # pip dependencies (for dependencies not available as RPM)
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+
 pip3 install --upgrade pip
+pip3 install setuptools setuptools-rust
```

Comment:
What are the setuptools and setuptools-rust needed for if we don't test AutoPy on this platform any longer?